dataactbroker/README.md

```diff
@@ -773,6 +773,153 @@ Example output if there are no files available:
 }
 ```
 
+# File Generation Routes
+
+## Generate Files
+**Route:** `/generate_file`
+
+**Method:** `POST`
+
+This route sends a request to the backend to utilize the relevant external APIs and generate the relevant file for the metadata that is submitted.
+
+**Deprecation Notice:** This route replaces `/generate_d1_file` and `/generate_d2_file`.
```

Comment:
Cool, thanks, I didn't even notice you'd left out the`V1`. It was smart for you to set up the versioning on the API. I created a post-launch story about setting up a process to notify any API users when we upgrade from V1 or make other changes that could impact people.